bciers/apps/reporting/src/app/components/attachments/AttachmentsPage.tsx

```diff
@@ -16,9 +21,21 @@ const AttachmentsPage: React.FC<HasReportVersion> = async ({ version_id }) => {
   const uploadedAttachmentsDict =
     getDictFromAttachmentArray(uploadedAttachments);
 
+  //🔍 Check if is a supplementary report
+  const isSupplementaryReportResponse =
+    await getIsSupplementaryReport(version_id);
+  const isSupplementaryReport =
+    isSupplementaryReportResponse.is_supplementary_report_version;
+  // Initialize confirmations variable
+  let initialSupplementaryConfirmation: SupplementaryConfirmation | undefined;
+  if (isSupplementaryReport) {
+    // Get attachment confirmations if it's a supplementary report
+    initialSupplementaryConfirmation =
+      await getAttachmentConfirmation(version_id);
+  }
+
```

Comment:
This code would go away if we have this already returned by the `getAttachments` call just above